File: test/fixtures/ruby/get_cookie_with_empty_value.rb
require 'net/http'

uri = URI('http://localhost:28139/cookies')
req = Net::HTTP::Get.new(uri)
req['accept'] = 'application/json'
req['Cookie'] = 'mysamplecookie=someValue; emptycookie=; otherCookie=2'

req_options = {
  use_ssl: uri.scheme == 'https'
}
res = Net::HTTP.start(uri.hostname, uri.port, req_options) do |http|
  http.request(req)
end

File: test/fixtures/java/get_cookie_with_empty_value.java
import java.io.IOException;
import java.net.URI;
import java.net.http.HttpClient;
import java.net.http.HttpRequest;
import java.net.http.HttpResponse;

HttpClient client = HttpClient.newHttpClient();

HttpRequest request = HttpRequest.newBuilder()
    .uri(URI.create("http://localhost:28139/cookies"))
    .GET()
    .setHeader("accept", "application/json")
    .setHeader("Cookie", "mysamplecookie=someValue; emptycookie=; otherCookie=2")
    .build();

HttpResponse<String> response = client.send(request, HttpResponse.BodyHandlers.ofString());
